server/controller/admin/user/userController.js

```diff
@@ -38,4 +36,98 @@ const getProfileDataOfAllExistingUser = async (req, res) => {
   }
 };
 
-module.exports = { getProfileDataOfAllExistingUser };
+const addEmployee = async (req, res) => {
+  const SECRET_KEY = process.env.SECRET_KEY;
+
+  try {
+    const {
+      name,
+      phone,
+      email,
+      personal_email,
```

Comment:
```suggestion
      workEmail,
      personalEmail,
```
Name it so it is distinguishable. And please follow the 'camel casing' convention for compound words.

also, **you probably forgot to put the address field**

Reply:
updated